### PR TITLE
Stitching: Solve quality issues of 360 and surround view stitching.

### PR DESCRIPTION
**Pull request by wujunkai166**
* Suggested output resolution for surround view is 1920*640.
 * test cmd line:
   ./test-image-stitching --input input0.nv12 --input input1.nv12   \
   --input input2.nv12 --input input3.nv12 --output output.mp4      \
   --input-w 1280 --input-h 800 --output-w 1920 --output-h 640      \
   --scale-mode local --enable-fisheyemap --res-mode 1080p4         \
   --surround-mode bowl --fm-ocl false --framerate 30.0 --save true \
   --fisheye-num 4 --all-in-one false

### CODE REVIEW

**Comment by wujunkai166** [comment]
@windyuan

File: modules/ocl/cv_feature_match.cpp [hunk 1]
```diff
@@ -182,7 +182,7 @@ CVFeatureMatch::detect_and_match (
     std::vector<uchar> status;
     std::vector<cv::Point2f> corner_left, corner_right;
     cv::Ptr<cv::Feature2D> fast_detector;
-    cv::Size win_size = cv::Size (41, 41);
+    cv::Size win_size = cv::Size (5, 5);
```
Comment (windyuan) [inline modules/ocl/cv_feature_match.cpp]: do we need change back to (5,5) for search window?

**Comment by wujunkai166** [comment]
@windyuan

**Comment by windyuan** [comment]
merged. thanks.

**Pull request merged**